app/javascript/packages/form-steps/form-steps.tsx

```diff
@@ -272,7 +279,7 @@ function FormSteps({
     const isComplete = nextStepIndex === steps.length;
     if (isComplete) {
       // Clear step parameter from URL.
-      setStepName(null);
+      setStepName(undefined);
```

Comment:
Consider how to handle this completeness behavior when using path-based history. Maybe just remove it?

Reply:
> Consider how to handle this completeness behavior when using path-based history. Maybe just remove it?

Removed in 4006ccd8f.